src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -186,6 +189,49 @@ const SimOptionsPage = () => {
         { name: 'hPa', setting: 'HPA' },
     ];
 
+    const mcduInputSetting = (
+        <div className="py-4 flex flex-row justify-between items-center">
+            <span>
+                <span className="text-lg text-gray-300">MCDU Keyboard Input</span>
+                <span className="text-lg text-gray-500 ml-2">(unrealistic)</span>
+            </span>
+            <Toggle value={mcduInput === 'ENABLED'} onToggle={(value) => setMcduInput(value ? 'ENABLED' : 'DISABLED')} />
+        </div>
+    );
+
+    const mcduTimeoutSetting = (mcduInput === 'ENABLED')
```

Comment:
**suggestion:** I would extract those to separate components

Reply:
solved by modifying SimpleInput to have proper logic for disabling <input>, a better solution than making another component.